src/main/target/CC3D/target.h

```diff
@@ -77,75 +79,103 @@
-#define NRF24_CE_GPIO                   GPIOA
-#define NRF24_CE_PIN                    GPIO_Pin_1
-#define NRF24_CE_GPIO_CLK_PERIPHERAL    RCC_APB2Periph_GPIOA
-#define NRF24_CSN_GPIO                  GPIOA
-#define NRF24_CSN_PIN                   GPIO_Pin_0
-#define NRF24_CSN_GPIO_CLK_PERIPHERAL   RCC_APB2Periph_GPIOA
-#define NRF24_SCK_GPIO                  GPIOB
-#define NRF24_SCK_PIN                   GPIO_Pin_5
-#define NRF24_MOSI_GPIO                 GPIOB
-#define NRF24_MOSI_PIN                  GPIO_Pin_1
-#define NRF24_MISO_GPIO                 GPIOB
-#define NRF24_MISO_PIN                  GPIO_Pin_0
-
-#define SERIAL_PORT_COUNT 3
-
+    #define DEFAULT_RX_FEATURE      FEATURE_RX_NRF24
+    #define DEFAULT_FEATURES        FEATURE_SOFTSPI
+    #define USE_RX_SYMA
+    //#define USE_RX_V202
+    #define USE_RX_CX10
+    //#define USE_RX_H8_3D
+    #define USE_RX_REF
+    #define NRF24_DEFAULT_PROTOCOL  NRF24RX_SYMA_X5C
+    //#define NRF24_DEFAULT_PROTOCOL NRF24RX_V202_1M
+    //#define NRF24_DEFAULT_PROTOCOL NRF24RX_H8_3D
+
+    #define USE_SOFTSPI
+    #define USE_NRF24_SOFTSPI
+
+    // RC pinouts
+    // RC1              GND
+    // RC2              power
+    // RC3  PB6/TIM4    unused
+    // RC4  PB5/TIM3    SCK / softserial1 TX / sonar trigger
+    // RC5  PB0/TIM3    MISO / softserial1 RX / sonar echo / RSSI ADC
+    // RC6  PB1/TIM3    MOSI / current
+    // RC7  PA0/TIM2    CSN / battery voltage
+    // RC8  PA1/TIM2    CE / RX_PPM
+
+    // Nordic Semiconductor uses 'CSN', STM uses 'NSS'
+    #define NRF24_CE_GPIO                   GPIOA
+    #define NRF24_CE_PIN                    GPIO_Pin_1
+    #define NRF24_CE_GPIO_CLK_PERIPHERAL    RCC_APB2Periph_GPIOA
+    #define NRF24_CSN_GPIO                  GPIOA
+    #define NRF24_CSN_PIN                   GPIO_Pin_0
+    #define NRF24_CSN_GPIO_CLK_PERIPHERAL   RCC_APB2Periph_GPIOA
+    #define NRF24_SCK_GPIO                  GPIOB
+    #define NRF24_SCK_PIN                   GPIO_Pin_5
+    #define NRF24_MOSI_GPIO                 GPIOB
+    #define NRF24_MOSI_PIN                  GPIO_Pin_1
+    #define NRF24_MISO_GPIO                 GPIOB
+    #define NRF24_MISO_PIN                  GPIO_Pin_0
+
+    #define SERIAL_PORT_COUNT 3
 #else
+    #define USE_SOFTSERIAL1
+    #define SERIAL_PORT_COUNT       4
 
-#define USE_SOFTSERIAL1
-#define SERIAL_PORT_COUNT       4
+    #ifdef USE_UART1_RX_DMA
+        #undef USE_UART1_RX_DMA
+    #endif
 
-#ifdef USE_UART1_RX_DMA
-#undef USE_UART1_RX_DMA
-#endif
+    #define SOFTSERIAL_1_TIMER      TIM3
+    #define SOFTSERIAL_1_TIMER_TX_HARDWARE 1 // PWM 2
+    #define SOFTSERIAL_1_TIMER_RX_HARDWARE 2 // PWM 3
 
-#define SOFTSERIAL_1_TIMER      TIM3
-#define SOFTSERIAL_1_TIMER_TX_HARDWARE 1 // PWM 2
-#define SOFTSERIAL_1_TIMER_RX_HARDWARE 2 // PWM 3
+    #define DEFAULT_RX_FEATURE FEATURE_RX_PPM
+#endif // USE_RX_NRF24
 
-#define DEFAULT_RX_FEATURE FEATURE_RX_PPM
 
-#endif // USE_RX_NRF24
+#if defined(CC3D_OPFLOW)
```

Comment:
I **really** like the CC3D.

For the reasons you say. And also because it is available in really small form factors.

I like to fly small brushed quads. I currently use MicroSciSky, but when I get around to it my plan is to used CC3D mini, NRF24 and some FETs.